app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -166,32 +167,17 @@ private fun HelpCenterHomeScreen(
           )
         }
         Spacer(Modifier.height(40.dp))
-        AnimatedVisibility(
-          visible = quickLinks.isNotEmpty(),
-          enter = fadeIn() + expandVertically(clip = false, expandFrom = Alignment.CenterVertically),
-          exit = fadeOut() + shrinkVertically(clip = false, shrinkTowards = Alignment.CenterVertically),
-        ) {
-          HelpCenterSection(
-            title = stringResource(R.string.HC_QUICK_ACTIONS_TITLE),
-            chipContainerColor = MaterialTheme.colorScheme.typeContainer,
-            contentColor = MaterialTheme.colorScheme.onTypeContainer,
-            content = {
-              Column(
-                verticalArrangement = Arrangement.spacedBy(4.dp),
-              ) {
-                for (quickLink in quickLinks) {
+        HelpCenterSection(
+          title = stringResource(R.string.HC_QUICK_ACTIONS_TITLE),
+          chipContainerColor = MaterialTheme.colorScheme.typeContainer,
+          contentColor = MaterialTheme.colorScheme.onTypeContainer,
+          content = {
```

Comment:
Now with the content here not being inside any sort of animation container, if we have those 5 entries shimmering here, and the real response comes back and we then only have 1 item this change will happen as well in a jumpy manner instead of smoothly animating to there.
Could we look into (with the sealed interface as discussed in the other parts of this PR) wrapping that `when` with `AnimatedVisibility` again perhaps? And then we can figure out the best way to animate between those two states. Or even the "no quick actions" state.